go_logging_server/logging/setup.go

```
@@ -0,0 +1,98 @@
+	var err error
+
+	//loops through files
+	for _, file := range files {
+
+		//create dir
+		err = utils.CreateDirectory(file.path)
+		if err != nil {
+			return err
+		}
+
+		file.file, err = createLogFile(file.filebaseName, file.path)
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)